src/components/CartList/CartList.tsx

```diff
@@ -0,0 +1,15 @@
+import VillaCard from "../VillaCard/VillaCard";
+import styles from "./CardList.module.css";
+import { useStore } from "../../store/store";
+
+export const CartList = () => {
+  const { villaBuyList } = useStore();
+
+  return (
+    <div className={styles.wishListContainer}>
+      {villaBuyList.map((item) => (
+        <VillaCard villa={item} />
```

Comment:
Add **key** here. Very important for React. Also check the console for errors/warnings